auth-web/src/components/auth/manage-business/SearchBusinessNameRequest.vue

```diff
@@ -177,10 +181,20 @@ export default class SearchBusinessNameRequest extends Vue {
   showAddNRModal () {
     this.$refs.addNRDialog.open()
   }
-  businessEvent (event: { name: string, identifier: string }) {
+  async businessEvent (event: { name: string, identifier: string }) {
     this.businessName = event?.name || ''
     this.businessIdentifier = event?.identifier || ''
-    this.showManageBusinessDialog = true
+    if (this.isGovStaffAccount) {
+      try {
+        let businessData: LoginPayload = { businessIdentifier: this.businessIdentifier }
+        await this.addBusiness(businessData)
+        this.$emit('add-success', this.businessIdentifier)
```

Comment:
TODO: Clear out the BusinessLookup search bar after the business has been added.
![image](https://github.com/bcgov/sbc-auth/assets/122643780/ed0e2514-b62a-4bd0-b9cb-54524f08c99e)